### PR TITLE
add multilingue(zh-CN/zh-TW) support

### PR DESCRIPTION
**Pull request by zx900930**
current progress: 100% ready for merge #38

### CODE REVIEW

File: app/js/lib/tooltip.js [hunk 1]
```diff
@@ -20,218 +21,185 @@ module.exports = {
...
 
-// <p>
+// '<p>'+i18next.t("
 // Examples:</br>
 // Force at least one: Speed > 10 for speed units.</br>
 // Force at least one: Hp % / Def % / Speed, for tanks.</br>
 // Force at least two: Atk / Atk % / Cr / Cd / Speed, for dps units.</br>
-// </p>
+// ")+'</p>'
 // `
```
Comment (fribbels) [inline app/js/lib/tooltip.js:38]: This part isn't needed anymore, will remove

File: app/locales/zh-TW/gridlocale.js [hunk 2]
```diff
@@ -0,0 +1,269 @@
+// Example locale file for English, give this to your locale team to translate
+
+const AG_GRID_LOCALE_ZH_TW = {
```
Comment (fribbels) [inline app/locales/zh-TW/gridlocale.js:3]: I wonder if there are predefined locale files.. seems like a waste to have every team translate their own file

File: app/locales/zh-TW/translation.json [hunk 3]
```diff
@@ -0,0 +1,821 @@
...
+    "Idol's Cheer": "歌姬助陣",
+    "Iron Fan": "鐵扇",
+    "Junkyard Dog": "Junkyard Dog",
+    "Justice for All": "為眾人的正義",
+    "Knowledge Seed": "知識種子",
+    "Last Teatime": "最後的下午茶",
+    "Manica of Control": "控制的護臂",
+    "Manifu": "瑪尼普",
+    "Merciless Glutton": "無情的大食客",
+    "Ms. Confille": "Miss Confile",
```
Comment (fribbels) [inline app/locales/zh-TW/translation.json:699]: Couple missing artifacts

File: app/locales/zh/translation.json [hunk 4]
```diff
@@ -0,0 +1,821 @@
...
+    "Idol's Cheer": "歌姬助阵",
+    "Iron Fan": "铁扇",
+    "Junkyard Dog": "Junkyard Dog",
+    "Justice for All": "为众人的正义",
+    "Knowledge Seed": "知识种子",
+    "Last Teatime": "最后的下午茶",
+    "Manica of Control": "控制的护臂",
+    "Manifu": "玛尼普",
+    "Merciless Glutton": "无情的大食客",
+    "Ms. Confille": "Miss Confile",
```
Comment (fribbels) [inline app/locales/zh/translation.json:699]: Couple missing artifacts

File: app/locales/en/translation.json [hunk 5]
```diff
@@ -0,0 +1,800 @@
+{
+ "Optimizer": "Optimizer",
+ "Ocr Debugger": "Ocr Debugger",
+ "Subprocess Debugger": "Subprocess Debugger",
+ "Clear language cache": "Clear language cache",
+ "Gear": "Gear",
+ "Heroes": "Heroes",
+ "Hero": "Hero",
+ "Start": "Start",
```
Comment (fribbels) [inline app/locales/en/translation.json:9]: Is the EN translation needed? Seems a bit redundant

File: app/js/lib/i18n/i18n.js [hunk 6]
```diff
@@ -0,0 +1,94 @@
+/* eslint-disable */
+window.i18next.use(window.i18nextChainedBackend).use(window.i18nextBrowserLanguageDetector).init({
+  debug: 'true',
```
Comment (fribbels) [inline app/js/lib/i18n/i18n.js:3]: Will clean this file up a bit for formatting

File: app/locales/zh-TW/translation.json (hunk 3 above)
Comment (zx900930) [reply app/locales/zh-TW/translation.json:699]: some artifact are not translated in the game like junkyard dog.

File: app/locales/en/translation.json (hunk 5 above)
Comment (zx900930) [reply app/locales/en/translation.json:9]: its not needed，but its an reference for other translators. they can copy it and do the translation

**Review by fribbels — APPROVED** [review]

**Pull request merged**